LetsDinnerV2 MessagesExtension/Extensions/String.swift

```diff
@@ -14,6 +14,8 @@ import Foundation
 // for example = FE
 // "" = ??
 
+let ACCEPTABLE_CHARACTERS = " ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_."
```

Comment:
Probably better to use CharacterSet from Swift. Why is this not respecting camelCase? Copy paste?

Reply:
i dunno, there should be a reason why people not using camel case

Reply:
No, it is a C or Java convention, not a Swift one

Reply:
https://nshipster.com/characterset/

Reply:
So you want to keep this ugly string ok